core-java-modules/core-java-security-4/src/test/java/com/baeldung/enablessldebug/SSLDebugLoggerUnitTest.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.enablessldebug;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+import org.junit.Test;
+
+public class SSLDebugLoggerUnitTest {
+    @Test
+    public void givenDebugging_whenSystemProperties_thenEnableSSL() {
+        SSLDebugLogger.enableSSLDebugUsingSystemProperties();
```

Comment:
To replicate the dynamic on-off behaviour of this approach, can you modify this test to

- Enable SSL debugging
- Assert it's enabled
- Disable SSL debugging
- Assert it's disabled

Reply:
Done